core/jobs/src/main/java/com/cognifide/aet/job/common/comparators/requestmonitoring/utils/RequestMonitoringResults.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * AET
+ * <p>
+ * Copyright (C) 2013 Cognifide Limited
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.cognifide.aet.job.common.comparators.requestmonitoring.utils;
+
+import java.util.SortedSet;
+import java.util.TreeSet;
+
+public class RequestMonitoringResults {
+
+  private final SortedSet<RequestMonitoringResult> results;
```

Comment:
I would change it's type to `Collection` if there are no other requirements which force using this specific interface. I don't think explicitly informing this collection saves order is necessary in this context.

Reply:
Done.